spec/unit/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'user'
+
+describe User do
+
+  describe '.create' do
+
```

Comment:
good tests, again using suitable matchers. Your `.create_new_instance` method creates a User object - could you test more explicitly that it does this?